assignment/assignment-bundles/resources/assignment.properties

```diff
@@ -1008,3 +1013,21 @@ grades.late=Late submission
 grades.lateness.late=Late
 grades.lateness.ontime=On time
 grades.lateness.unknown=Unknown
+
+cr.notprocess.warning = You have uploaded a file with an extension that will not be processed by the content review service
+cr.size.warning = You have uploaded a file which size is too big for the content review service
+submission.inline=Inline Submission
+content.review=Content Review
+content.review.inbox=Assignment Inbox
+review.oldsite=This site was created more than {0} years ago and there's a risk the content review site might have expired.
+review.sitechars=The title of this site is shorter than the minimum allowed by the content review service. If you wish to route submissions through {0} then you need to choose a site title with more than {1} characters.
+review.sitecharslong=The title of this site is longer than the maximum allowed by the content review service. If you wish to route submissions through {0} then you need to choose a site title with less than {1} characters.
+review.assignchars=The title of this assignment is shorter than the minimum allowed by the content review service. If you wish to route submissions through {0} then you need to choose a title with more than {1} characters.
+review.assigncharslong=The title of this assignment is longer than the maximum allowed by the content review service. If you wish to route submissions through {0} then you need to choose a title with less than {1} characters.
+review.originality.alt=View originality report: score
+review.user.email=The content review service requires your email to be set. Your submission cannot be reviewed by {0} until you update your details.
+review.user.name=The content review service requires your first name and surname to be set. Your submission cannot be reviewed by {0} until you update your details.
+review.user.lastname=The content review service requires your surname to be set. Your submission cannot be reviewed by {0} until you update your details.
+review.instructor.fields=The content review service requires your first name, surname and email must to be set. If you wish to route submissions through {0} then you need to update your details.
+review.conflicting.popup=This item has been graded in both Turnitin and in the Assignments Tool. The mark set in the Assignments Tool is the one that is displayed here.
+allowResubmission.review.warning=Any submissions made after the Assignment Due Date will not be sent to {0}
```

Comment:
A few grammar revisions could be made in some of these. Not sure if regular users know what "content review service" means, I'd just put {0} twice (Ie. 'Turnitin').